pkg/status/render.go

```diff
@@ -28,6 +28,27 @@ func init() {
 	templateFolder = filepath.Join(common.GetDistPath(), "templates")
 }
 
+// FormatRunnerStats changes to structure of runner stats to be displayed on the status pages
```

Comment:
Could we directly correctly format the runner stats in the runners ?

Reply:
This will change the format of the data exposed through expvar, I didn't want to have any impact on this

Reply:
We discussed this and it's okay to change the structure

Reply:
I tried to change the structure of the runnerStats from map[checkID]interface{} to map[checkName][checkID]interface{} but this is not very beautiful and it induces some additional difficulties on the templating sides so that I still need a formating function. That's why I would prefer to keep the changes as is.